plugin/src/main/java/at/helpch/chatchat/ChatChatPlugin.java

```diff
@@ -128,19 +127,21 @@ private void registerCommands() {
                 .collect(Collectors.toList())));
 
         List.of(
-            new MainCommand(),
-            new ReloadCommand(this),
-            new WhisperCommand(this, false),
-            new ReplyCommand(this, new WhisperCommand(this, true)),
-            new WhisperToggleCommand(this),
-            new SocialSpyCommand(this)
+                new MainCommand(),
+                new ReloadCommand(this),
+                new WhisperCommand(this, false),
+                new ReplyCommand(this, new WhisperCommand(this, true)),
+                new WhisperToggleCommand(this),
+                new SocialSpyCommand(this)
         ).forEach(commandManager::registerCommand);
 
         // register channel commands
         configManager.channels().channels().values().stream()
-                .map(Channel::commandName) // don't register empty command names
+                .map(Channel::commandNames) // don't register empty command names
                 .filter(s -> !s.isEmpty())
-                .map(command -> new SwitchChannelCommand(this, command))
-                .forEach(commandManager::registerCommand);
+                .forEach(commandNames -> commandNames.stream()
+                        .map(commandName -> new SwitchChannelCommand(this, commandName))
```

Comment:
I already had a these changes made I just completely forgot to push and open a PR. Anyways I was thinking make the first command a main command then the rest in that list aliases instead of making a new command for each string? We can pass the aliases in the BaseCommand constructor as a list or an array of strings (I forgor.)

Idk if there's any actual benefit for doing that. Most likely just more clashing between the aliases and the commands from other plugins now that I think about it.

Reply:
I can go ahead and do that, looks like I just need to modify the constructor for SwitchChannelCommand a bit so I can pass aliases to the BaseCommand